core/src/main/scala/org/apache/spark/executor/CoarseGrainedExecutorBackend.scala

```diff
@@ -98,7 +98,7 @@ private[spark] class CoarseGrainedExecutorBackend(
       if (executor ne null) {
         logDebug("Got assigned tasks " + tasks.map(_.taskId).mkString(","))
         for (task <- tasks) {
-          logInfo("Got assigned task " + task.taskId)
+          logDebug("Got assigned task " + task.taskId)
```

Comment:
The same taskLogger can be used above too. Perhaps the instance can be held in the Task object instead of SparkContext (i.e. object Task can just extend Logging) so that it can be used across modules more naturally.